services/minespace-web/src/components/dashboard/mine/overview/Overview.js

```diff
@@ -0,0 +1,152 @@
+// TODO: Remove this when the file is more fully implemented.
+/* eslint-disable */
+
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import CustomPropTypes from "@/customPropTypes";
+import { Row, Col, Card, Descriptions, Typography, Icon } from "antd";
+import * as Strings from "@/constants/strings";
+import * as Contacts from "@/constants/ministryContacts";
+
+const { Paragraph, Text, Title } = Typography;
+
+const propTypes = {
+  mine: CustomPropTypes.mine.isRequired,
+};
+
+const defaultProps = {};
+
+const ContactCardRow = (props) => (
+  <Row gutter={[32, 16]}>
+    <Col span={3}>
+      <Icon type={props.icon} style={{ fontSize: "2em" }} />
+    </Col>
+    <Col span={20} offset={1}>
+      <Text strong style={{ textTransform: "uppercase" }}>
+        {props.label}
+      </Text>
+      <br />
+      <Text>{props.value}</Text>
+    </Col>
+  </Row>
+);
+
+const ContactCard = (props) => (
```

Comment:
I forgot to do that when I did it for the TableSummaryCard component. I'll do that in another PR or in this one tomorrow morning! Can't merge it in because it's failing to build.